tests/snapshot/test_snapshot.py

```diff
@@ -12,11 +11,6 @@
 
 class SnapshotOperationTest(NodeProvider, unittest.TestCase):
 
-    PATH_DATA = 'data_test_snapshot_ops'
-
-    def tearDown(self):
```

Comment:
see my comment above, would be great to keep that.

Reply:
I couldn't use it as it's removes specific single folder. I added `tearDown` of the base class instead in finally block as it takes care about folders created by `mkdtemp`

Reply:
removed teardown from finally - thx for the hint about automatic call